index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+    <div class="header-content">
+      <div class="logo">
+        <a href="#">Daniel</a>
+      </div>
+        <nav>
+          <img class="hamburger" src="images/menu.svg" alt="menu">
```

Comment:
Kindly, fix the `src` of the image currently it's referencing the wrong directory and the result showing on the page is like this 👇 

<img width="320" alt="image" src="https://user-images.githubusercontent.com/34578631/170542319-4b721429-a206-48c1-bdc1-7e55adc7590a.png">